src/mame/mr/dribling.cpp

```diff
@@ -453,9 +501,34 @@ void dribling_state::dribling(machine_config &config)
 	PALETTE(config, "palette", FUNC(dribling_state::palette), 256);
 
 	// sound hardware
-}
+	SPEAKER(config, "mono").front_center();
+
+	NETLIST_SOUND(config, "snd_nl", 12500)
+		.set_source(NETLIST_NAME(dribling))
+		.add_route(ALL_OUTPUTS, "mono", 1.0);
+
+	NETLIST_LOGIC_INPUT(config, "snd_nl:pb0",  "I_PB0.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:pb1",  "I_PB1.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:pb2",  "I_PB2.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:pb3",  "I_PB3.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:pb4",  "I_PB4.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:pb5",  "I_PB5.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:pb6",  "I_PB6.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:pb7",  "I_PB7.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:folla_b", "I_FOLLA_B.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:folla_m", "I_FOLLA_M.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:folla_a", "I_FOLLA_A.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:calcio_a","I_CALCIO_A.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:fischio",  "I_FISCHIO.IN",0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:calcio_b",  "I_CALCIO_B.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:contrasto", "I_CONTRASTO.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:stop_palla","I_STOP_PALLA.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:parata","I_PARATA.IN", 0);
+	NETLIST_LOGIC_INPUT(config, "snd_nl:enable",  "ENABLE_SOUND.IN",0);
+	NETLIST_STREAM_OUTPUT(config, "snd_nl:cout0", 0, "OUTPUT").set_mult_offset(1.0, 0);
 
 
+}
 
 /*************************************
```

Comment:
There’s still the diff noise here from adding an extra blank line inside the function and getting rid of one from the two blank lines separating functions.